ta/arch/arm/ta.ld.S

```diff
@@ -53,7 +53,15 @@ SECTIONS {
 	.ARM.exidx : { *(.ARM.exidx* .gnu.linkonce.armexidx.*) }
         .ctors : { *(.ctors) }
         .dtors : { *(.dtors) }
-	.got : { *(.got.plt) *(.got) }
+	.dynamic : { *(.dynamic) } :dyn :rodata
```

Comment:
If you move `.dynamic` to `rwdata` below (where it belongs), then all the custom flags in PHDR can be removed (and therefore the big ugly comment, too).

I suspect the linker script could be vastly simplified, but that's another topic.